lib/fantasy_football_nerd.rb

```diff
@@ -84,7 +84,6 @@ def self.weekly_rankings(position, week = nil)
 
   def self.weekly_projections(position, week = nil)
     #FFNerd defaults to current week if week is left blank
-    raise "Weekly projections don't include DEF (but you can find those values in weekly rankings)" if position == "DEF"
```

Comment:
Great thanks! Would you mind adding a test for that weather method and then I will merge this in.